homeassistant/components/netgear/__init__.py

```diff
@@ -18,6 +22,21 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     except CannotLoginException as ex:
         raise ConfigEntryNotReady from ex
 
+    port = entry.data.get(CONF_PORT)
+    ssl = entry.data.get(CONF_SSL)
+    if port != router.port or ssl != router.ssl:
+        data = entry.data.copy()
```

Comment:
```py
data = {**entry.data, CONF_PORT: router.port, CONF_SSL: router.ssl}
```

Reply:
corrected in PR https://github.com/home-assistant/core/pull/64428